src/platform/Linux/bluez/BluezEndpoint.cpp

```diff
@@ -155,7 +154,6 @@ gboolean BluezEndpoint::BluezCharacteristicAcquireNotify(BluezGattCharacteristic
 #endif // CHIP_ERROR_LOGGING
     BluezConnection * conn       = nullptr;
     bool isAdditionalAdvertising = false;
-    GAutoPtr<GVariant> option_mtu;
```

Comment:
was this just unused previously?

Reply:
Yes, this is unused variable. However, it is basically `std::unique_ptr` so there is no warning because destructor is called...